### PR TITLE
Proxy containers: using a custom container image for a service

### PR DESCRIPTION
**Pull request by avshiliaev**
# Description

Describes how to use a container image from a different registry path for the containerized proxy.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3

# Links

* https://github.com/SUSE/spacewalk/issues/19610

### CODE REVIEW

File: modules/installation-and-upgrade/pages/proxy-container-installation.adoc [hunk 1]
```diff
@@ -109,6 +109,29 @@ In this file it is possible to modify the tag to use for the container images:
 
 - [literal]``TAG``=[path]``latest``
 
+=== Using a custom container image for a service
+
+By default, the {productname} Proxy suite is set to use the same image version and registry path for each of its services. However, there's a possibility to change this behavior by overriding the default values for a specific service. The [literal]``uyuni-proxy`` CLI bundled with the package, has an [literal]``update image`` command taking the following arguments:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-container-installation.adoc:114]:
```suggestion
By default, the {productname} Proxy suite is set to use the same image version and registry path for each of its services.
However, it is possible to override the default values for a specific service.
The [literal]``uyuni-proxy`` CLI bundled with the package, has an [literal]``update image`` command  with the following parameters:
```

File: modules/installation-and-upgrade/pages/proxy-container-installation.adoc [hunk 2]
```diff
@@ -109,6 +109,29 @@ In this file it is possible to modify the tag to use for the container images:
...
+
+- [literal]``-s`` for the service name
+- [literal]``-t`` for the version tag
+- [literal]``-r`` for the registry path
+
+An example usage would look like this:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-container-installation.adoc:120]:
```suggestion
For example, use it like this:

```

File: modules/installation-and-upgrade/pages/proxy-container-installation.adoc [hunk 3]
```diff
@@ -109,6 +109,29 @@ In this file it is possible to modify the tag to use for the container images:
...
+----
+uyuni-proxy update image -s httpd -t 0.1.0 -r registry.opensuse.org/uyuni
+----
+It would adjust a configuration file for the httpd service, taking [path]``registry.opensuse.org/uyuni`` as the registry and [literal]``0.1.0`` as the version tag and then restart it.
+
+To reset the values to defaults, run the proxy reset command, specifying the service with the -s flag. An example usage would look like the following:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-container-installation.adoc:126]:
```suggestion
To reset the values to defaults, run the proxy reset command, specifying the service with the [literal]``-s`` parameter.
For example, use it like this:
```

File: modules/installation-and-upgrade/pages/proxy-container-installation.adoc [hunk 4]
```diff
@@ -109,6 +109,29 @@ In this file it is possible to modify the tag to use for the container images:
...
+
+----
+uyuni-proxy reset -s httpd
+----
+
+This command would reset the configuration of the [literal]``httpd`` service to the global defaults and restarts it.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-container-installation.adoc:132]:
```suggestion
This command would reset the configuration of the [literal]``httpd`` service to the global defaults and then restart it.
```

File: modules/installation-and-upgrade/pages/proxy-container-installation.adoc [hunk 5]
```diff
@@ -109,6 +109,29 @@ In this file it is possible to modify the tag to use for the container images:
...
+
+See the [literal]``uyuni-proxy --help`` for more detail.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-container-installation.adoc:134]:
```suggestion
For more information, see the [literal]``uyuni-proxy --help``.
```

**Review by keichwa — CHANGES_REQUESTED** [review]
Thanks for your PR!  Just a few cosmetic suggestions to make it consistent with our existing documentation.  If you agree please commit the suggestions.

**Comment by avshiliaev** [comment]
@keichwa Thank you! I totally agree with the changes, have commited all of them 😄

**Review by keichwa — APPROVED** [review]

**Pull request merged**